DataFormats/MuonReco/interface/Muon.h

```diff
@@ -288,7 +327,8 @@ namespace reco {
     /// get pointers to best segment and corresponding chamber in vector of chambers
     std::pair<const MuonChamberMatch*,const MuonSegmentMatch*> pair( const std::vector<const MuonChamberMatch*> &,
 									ArbitrationType type = SegmentAndTrackArbitration ) const;
-     
+    /// selector bitmap
+    unsigned int selectors_;
```

Comment:
is it safe to commit to 32 bits now, considering that 22 are already taken?
uint64_t could save from some headache in not so distant future.

Reply:
32 is good enough. We need to resist a temptation to introduce any minor modification as a separate selector. If that's done I don't think we will reach the limit any time soon. In other words it's good to have some limits to avoid id proliferation.

Reply:
Well, OK.
It's not obvious what happens with phase-2 specific IDs.
It's almost zero effort now to leave more room.